pkgs/by-name/ko/koboldcpp/package.nix

```
@@ -0,0 +1,115 @@
+{ lib
+, fetchFromGitHub
+, stdenv
+, autoPatchelfHook
+, makeWrapper
+, python311Packages
+, tk
+
+, koboldLiteSupport ? true
+
+, config
+, cudaPackages ? { }
+
+, openblasSupport ? !stdenv.isDarwin
+, openblas
+
+, cublasSupport ? config.cudaSupport
+
+, clblastSupport ? stdenv.isLinux
+, clblast
+, ocl-icd
+
+, vulkanSupport ? true
+, vulkan-loader
+
+, metalSupport ? stdenv.isDarwin && stdenv.isAarch64
+
+, march ? ""
+, mtune ? ""
+}:
+
+let
+  inherit (lib) optionals optionalString;
+
+  pythonInputs = with python311Packages; [
+    tkinter
+    customtkinter
+    packaging
+    tk
+  ];
+
+  cudaBuildInputs = with cudaPackages; [
+      libcublas
+  ];
```

```suggestion
  pythonInputs = builtins.attrValues { inherit (python311Packages) tkinter customtkinter packaging; };

  cudaBuildInputs = builtins.attrValues { inherit (cudaPackages) cudaPackages libcublas; };
```

https://nix.dev/guides/best-practices#with-scopes

There is no `python311Packages.tk`?